scriptworker/test/test_gpg.py

```diff
@@ -692,7 +599,15 @@ def test_verify_signed_git_commit_output_exception(output):
 
 
 @pytest.mark.asyncio
-async def test_get_git_revision_exception(mocker):
+async def test_get_latest_tag():
+    parent_dir = os.path.dirname(__file__)
+    expected = subprocess.check_output(["git", "describe", "--abbrev=0"], cwd=parent_dir)
```

Comment:
The test suite currently uses the current scriptworker repo as the testing ground, so no.  I did add a TODO above that says we should create a test git repo tarball with deterministic results.